src/core/adapters/gitlab.test.js

```diff
@@ -3,6 +3,18 @@ import { JSDOM } from 'jsdom';
 import scan from './gitlab';
 
 const ISSUEPAGE = `
+  <body data-page-type-id="22578">
+    <div class="content">
+      <div class="issue-details">
+        <div class="detail-page-description">
+          <h2 class="title">A Random GitLab Issue</h2>
+        </div>
+      </div>
+    </div>
+  </body>
+`;
+
+const LEGACYISSUE = `
```

Comment:
I couldn't name this `LEGACYISSUEPAGE`. It seemed like things started breaking after a certain name length 🤷‍♂️